src/components/cookie-banner/index.md.njk

```diff
@@ -17,121 +17,160 @@ Allow users to accept or reject cookies which are not essential to making your s
 
 ## When to use this component
 
-You must tell users about any cookies you’re using - or any similar technologies that store files on the user’s device, like HTML5 local storage or service workers. You must get the user’s consent before setting any cookies which are not essential to providing the service.
+You must tell users about the cookies your service sets on their device.
```

Comment:
Does this match with the heading title (ie tell users when they should use the component), or is it more like a reminder of their legal obligations? If the latter, have we covered this in the 'Before you start' section, just below? That mentions GDPR etc and tells users to check with their organisation's privacy expert.

Maybe we could change the first couple of sentences to one sentence with bullets? To something like:

> Use this component when you need to:
> - tell users about the cookies your service sets on their device
> - let users accept or reject any cookies that are not essential to providing your service